### PR TITLE
Add utilization card to Host dashboard

### PR DESCRIPTION
**Pull request by rawagner**
depends on https://github.com/openshift/console/pull/1732 and https://github.com/openshift/console/pull/1790

@honza 

host-util

### CODE REVIEW

**Comment by rawagner** [comment]
rebased, @jtomasek @honza please review

File: frontend/packages/metal3-plugin/src/components/dashboard/queries.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+export enum HostQuery {
+  CPU_UTILIZATION = 'CPU_UTILIZATION',
+  MEMORY_UTILIZATION = 'MEMORY_UTILIZATION',
+  STORAGE_UTILIZATION = 'STORAGE_UTILIZATION',
+  NETWORK_IN_UTILIZATION = 'NETWORK_IN_UTILIZATION',
+  NETWORK_OUT_UTILIZATION = 'NETWORK_OUT_UTILIZATION',
+  NUMER_OF_PODS = 'NUMER_OF_PODS',
```
Comment (jtomasek) [inline frontend/packages/metal3-plugin/src/components/dashboard/queries.ts]: Is this a typo? `NUMBER_OF_PODS` ?

Comment (rawagner) [reply frontend/packages/metal3-plugin/src/components/dashboard/queries.ts]: yes, it is. Fixed

**Comment by rawagner** [comment]
> Probably not caused by this change but when changing the screen width, the font size of the labels change. Is that expected?

hm, i think they are scaling in size together with chart. Not sure if thats what we want..we can take a look at it later.

**Pull request merged**